packages/shared-lib/src/package-manager/helpers.ts

```diff
@@ -0,0 +1,61 @@
+import { ExpectedPackage } from './package'
+
+// Note: These functions are copied from Package Manager
+
+type Steps = Required<ExpectedPackage.ExpectedPackageHtmlTemplate['version']>['steps']
+
+export function htmlTemplateGetSteps(version: ExpectedPackage.ExpectedPackageHtmlTemplate['version']): Steps {
```

Comment:
I don't like these casparcg steps being duplicated from package-manager. but maybe that is unavoidable?